Added agent to deploy static website to AWS S3+Cloundfront

gpavlov2016 opened the pull request:
This agent automates the deployment of static websites (such as Webflow exports) to AWS S3 and CloudFront. 
It streamlines the process of hosting static content on AWS, ensuring proper configuration, cache invalidation, and deployment validation.

SagiMedina commented:
@gpavlov2016 Thanks for sharing! This is great!
A couple of questions before we merge it:

1. Is the exit_expression necessary here? (if so, please update the value)

> [The condition determines whether the agent run succeeded or failed. ](https://docs.qodo.ai/qodo-documentation/qodo-command/features/creating-and-managing-agents#id-2.-customize-your-agent)
> string ([JSONPath](https://jsonpath.com/))

2. The bucket argument is marked as required in the YAML file, but optional in the
TOML file. Which one is true?

gpavlov2016 commented:
Thanks for pointing that out @SagiMedina, I updated the configuration to address theses issues:
- bucket is optional in both yaml and toml
- removed success criteria
